J/JSON_C/build_tarballs.jl

```diff
@@ -0,0 +1,36 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "JSON_C"
+version = v"0.15.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://s3.amazonaws.com/json-c_releases/releases/json-c-0.15.tar.gz", "b8d80a1ddb718b3ba7492916237bbf86609e9709fb007e7f7d4322f02341a4c6")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd json-c-0.15
+cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release
+make install
+exit
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libjson-c", :JSON_C)
```

Comment:
Out of curiosity, why not
```suggestion
    LibraryProduct("libjson-c", :libjson_c)
```
?

Reply:
It was the little they listed on their readme https://github.com/json-c/json-c. I could change it to libjson-c. What does lib mean?

Reply:
"lib" stands for "library" :slightly_smiling_face:

Reply:
Ok, I'll change it